recipe/conda_build_config.yaml

```diff
@@ -67,11 +67,6 @@ hdf5:
   - 1.10               # [not ppc64le]
 icu:
   - 58                 # [not ppc64le]
-ipython:
```

Comment:
Removing this is fine with me, but we might need to add it back if anyone ever remembers or encounters why it was pinned here.